components/omega/doc/design/OmegaV1GoverningEqns.md

```diff
@@ -0,0 +1,741 @@
+
+### Momentum Forcing
+The discretized momentum forcing ${ \bf F}^u_{e,k}$ may include:
+
+#### Wind Forcing
+
+The wind forcing is applied as a top boundary condition during implicit vertical mixing as
+
+$$
+\frac{\tau_{e}}{[ h_{i,k}]_e}
+$$
+
+where $\tau$ is the wind stress in Pa. Since the mass-thickness $h$ is in kg/s/m$^2$, this results in the desired units of m/s$^2$ for a momentum tendency term.
+
+#### Bottom Drag
+
+Bottom Drag is applied as a bottom boundary condition during implicit vertical mixing as
+
+$$
+- C_D \frac{u_{e,k}\left|u_{e,k}\right|}{[v_{i,k}h_{i,k}]_e} .
+$$ (discrete-mom-bottom)
+
+The units of specific volume times mass-thickness $vh$ are length (m), so that the full term has units of m/s$^2$.
+
+#### Rayleigh Drag
+
+Rayleigh drag is a simple drag applied to every cell.  It is used to ensure stability during spin-up.
+
+$$
+- Ra \, u_{e,k}
+$$ (discrete-mom-Ra)
+
+### Tracer Diffusion
+
+The discretized tracer diffusion $ D^\varphi_{i,k}$ may include these terms, which are detailed below. Here $\kappa_2$ and $\kappa_4$ are written in front of the operator for simplicity.
+
+$$
+D^\varphi_{i,k} =  \kappa_2 \nabla^2 \varphi_{i,k} - \kappa_4 \nabla^4 \varphi_{i,k} +
+\frac{\partial }{\partial z} \left( \kappa_v \frac{\partial \varphi_{i,k}}{\partial z} \right)
+$$ (discrete-tracer-diff)
+
+#### Laplacian diffusion (del2)
+The Laplacian may be written as the divergence of the gradient,
+
+$$
+ h_{i,k} \nabla \cdot \left( \kappa_{2,e,k} \nabla \varphi_{i,k} \right).
+$$ (discrete-tracer-del2)
+
+See [Omega V0 Section 3.3.2](OmegaV0ShallowWater.md#del2-tracer-diffusion) for details of this calculation.
+
+#### Biharmonic diffusion (del4)
+The biharmonic is a Laplacian operator applied twice,
+
+$$
+ - h_{i,k} \nabla \cdot \left( \kappa_{4,e,k} \nabla
+\right[
+\nabla \cdot \left(  \nabla \varphi_{i,k} \right)
+\left]
+ \right).
+$$ (discrete-tracer-del4)
+
+Each of these operators are written as horizontal stencils in the [Omega V0 Operator Formulation Section](OmegaV0ShallowWater.md#operator-formulation)
+
+#### Vertical tracer diffusion
+As discussed above in the [momentum section](#vertical-momentum-diffusion), vertical derivatives may be written in terms of $z$ or $p$,
+
+$$
+\frac{\partial }{\partial z} \left( \kappa_v \frac{\partial {\bf \varphi}}{\partial z} \right)
+= \rho g^2 \frac{\partial }{\partial p} \left( \kappa_v \rho \frac{\partial {\bf \varphi}}{\partial p} \right)
+$$ (discrete-tracer-vertdiff)
+and $z$ is chosen. The second derivative stencil is
+
+$$
+h_{i,k} \frac{\partial }{\partial z} \left( \kappa_v \frac{\partial \varphi_{i,k}}{\partial z} \right)
+=
+\frac{h_{i,k}}{z_{i,k}^{top} - z_{i,k+1}^{top}} \left(
+\kappa_{i,k}^{top}
+\frac{\varphi_{i,k-1} - \varphi_k }{z_{i,k-1}^{mid} - z_k^{mid}}
```

Comment:
same comment as above, why just `top` on `kappa`?

Reply:
Changed all to top.